src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -98,20 +92,26 @@
 
   <xsl:template match="paramref">
     <xsl:if test="normalize-space(@name)">
-      <span class="paramref">
+      <code data-dev-comment-type="paramref">
```

Comment:
@vicancy  we'd better transfer the element into span with specific class in our javascript?